Remove Fedora reference (fedora-loadmodules.service) from qubes-db.service systemd unit file
--- a/daemon/qubes-db.service
+++ b/daemon/qubes-db.service
@@ -1,6 +1,6 @@
 [Unit]
 Description=Qubes DB agent
-After=systemd-modules-load.service fedora-loadmodules.service
+After=systemd-modules-load.service
 DefaultDependencies=no
 
 [Service]
